cinema/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from .views import movie_list, movie_detail
+
+urlpatterns = [
+    path("movie/", movie_list, name="movie-list"),
```

Comment:
I think we need to write our endpoint in the plural ("movies/", "movies/<int:pk>/")

Reply:
Done